menpofit/io.py

```diff
@@ -19,6 +21,71 @@
 MENPOFIT_BINARY_VERSION = 0
 
 
+def image_greyscale_rescale_preprocess(image, pointcloud, proportion_thresh=0.1,
+                                       crop_proportion=0.3, diagonal=200):
+    r"""
+    Pre-processing function for a given test image. The function does the
+    following:
+
+    1. If image is RGB, convert it to greyscale.
+    2. If the proportion of the pointcloud's size with respect to the image's
+       size is too small, then crop the image.
+    3. If the diagonal of the pointcloud's bounding box is too large,
+       then rescale the image.
+
+    The method returns a pre-processed copy of the input image and the total
+    transform object that was applied on it.
```

Comment:
You'd have to doc that users need to leave a `__pointcloud` key of landmarks with the current approach, with the suggested alternative, you don't